e2e-tests/.eslintrc.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
     'import/no-named-as-default': 'off',
     'implicit-arrow-linebreak': 'off',
     'import/prefer-default-export': 'off',
-    'import/no-extraneous-dependencies': ['error', { devDependencies: false }],
+    'import/no-extraneous-dependencies': ['error', { devDependencies: true }],
```

Comment:
```suggestion
    'import/no-extraneous-dependencies': ['error'],
```
Is `devDependencies: true` not the default?